### PR TITLE
Add auto-build docs

### PR DESCRIPTION
**Pull request by Stevent-fei**
### Describe what this PR does / why we need it
Add Github Action auto-build docs

### CODE REVIEW

**Comment by fanux** [comment]
What the output CloudImage name? kubernetes:[version] like kubernetes:v1.19.8?

**Comment by Stevent-fei** [comment]
> 

In order to distinguish between the basic image and the automatically built image, the suffix - Auto, for example: kubernetes: v1.19.8-auto

File: docs/design/auto-build.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
+# auto-build
```
Comment (kakaZhou719) [inline docs/design/auto-build.md:1]: upper case for title

File: docs/design/auto-build.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+For example:
+
+```shell
+kubernetes:v1.20.14-auto
+```
+
+After inputting, GitHub action will start to execute the automatic construction of the basic image of the corresponding version of kubernetes.
+
+When `kubernetes: version-auto` appears in the comment area, it indicates that the image construction is completed.
```
Comment (kakaZhou719) [inline docs/design/auto-build.md:42]: use proper nouns： "image build" not "image construction"

**Comment by bxy4543** [comment]
Since Auto-Build cannot be triggered by ordinary users, I think we should describe how to design it.

File: docs/design/auto-build.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+```shell
+/imagebuild 1.20.14 amd64
+```
+
+When `Image built successfully : kubernetes: version-auto` appears in the comment area, it indicates that the image build is completed.
+
+Image name:
+
+```shell
+kubernetes:v1.20.14-auto
```
Comment (fanux) [inline docs/design/auto-build.md:35]: kubernetes:v1.20.14-auto -> kubernetes:v1.20.14

Comment (Stevent-fei) [reply docs/design/auto-build.md:35]: If the version number is changed directly, it will be confused with the basic image

**Review by fanux — APPROVED** [review]

**Pull request merged**